### PR TITLE
fix: Update last criterion on MFA tools page

### LINKED ISSUE #2608: Regarding the last criterion on the MFA tools page
### Affected page

https://www.privacyguides.org/en/multi-factor-authentication/#criteria

### Description

Another person on the PG Matrix pointed this out:

> i just read https://www.privacyguides.org/en/multi-factor-authentication/ and it seems like Ente is not fit for the criteria?

     Auth provides end-to-end encrypted cloud backups so that you don't have to worry about losing your tokens.
     We use the same protocols Ente Photos uses to encrypt and preserve your data.

> did this change recently?

> not even optional is allowed, as per criteria

     Must not sync to a third-party cloud sync/backup service.

     Optional E2EE sync support with OS-native tools is acceptable, e.g. encrypted sync via iCloud.

> their service is third party (that's why it needs a login)

### Sources

- https://apps.apple.com/app/id6444121398 (source for the first quote in plain text, which can be found under the "SECURE BACKUPS" heading in the app's description)
- https://github.com/ente-io/ente/tree/main/auth#-architecture

### Before submitting

- [X] I am reporting something that is verifiably incorrect, not a suggestion or opinion.
- [X] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### PR DESCRIPTION
**Pull request by redoomed1**
Changes proposed in this PR:

- Update the last criterion on the Multi-Factor Authentication page (closes #2608)
  - In a similar fashion to #2579, the current wording regarding sync would disqualify Ente Auth, which is not at all ideal, considering the recent fiasco with Raivo. Looking at #2251, it looks like this criterion is a vestige of the now-removed Raivo recommendation. I can't access the [discussions](https://github.com/privacyguides/privacyguides.org/pull/1980#issuecomment-1431294392) that led to the [creation of this criterion](https://github.com/privacyguides/privacyguides.org/pull/1980); hopefully, this update keeps the spirit of the original criterion intact.
- Update links for Qubes ProxyVM guides as some of these docs have been migrated to the Qubes [forum](https://forum.qubes-os.org/t/announcement-qubes-community-project-has-been-migrated-to-the-forum/20367/)
- Update internal link for Nitrokey so that it points to the new Security Keys page

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

**Review by jonaharagon — APPROVED** [review]

**Comment by jonaharagon** [comment]
Hm, this is an interesting point. I think that @rollsicecream was correct in https://github.com/privacyguides/privacyguides.org/issues/2608#issuecomment-2144283326 that this criteria doesn't affect Ente Auth, but I can imagine that "third-party" is not very clear. The changes here sound easier to understand 👍

File: docs/multi-factor-authentication.md [hunk 1]
```diff
@@ -72,5 +72,5 @@ We highly recommend that you use mobile TOTP apps instead of desktop alternative
 
 - Source code must be publicly available.
 - Must not require internet connectivity.
-- Must not sync to a third-party cloud sync/backup service.
-    - **Optional** E2EE sync support with OS-native tools is acceptable, e.g. encrypted sync via iCloud.
+- Must not sync to a third-party cloud sync/backup service by default.
+    - **Optional** E2EE sync support is acceptable.
```
Comment (jonaharagon) [inline docs/multi-factor-authentication.md:76]:
```suggestion
- Cloud syncing must be optional, and (if available) sync functionality must be E2EE.
```

**Review by jonaharagon — COMMENTED** [review]
What about this?

**Comment by redoomed1** [comment]
I agree that the "third-party" wording is not very clear: is the first party the service provider (Ente, in this case) or the operating system?

I like your suggested change because it removes this question from consideration.

**Review by freddy-m — APPROVED** [review]

**Review by dngray — APPROVED** [review]

**Pull request merged**